packit/config/common_package_config.py

```diff
@@ -0,0 +1,134 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""
+Common package config attributes so they can be imported both in PackageConfig and JobConfig
+"""
+from typing import Optional, List, Union, Dict
+
+from packit.actions import ActionName
+from packit.config.sync_files_config import SyncFilesConfig
+from packit.config.notifications import (
+    NotificationsConfig,
+    PullRequestNotificationsConfig,
+)
+from packit.constants import PROD_DISTGIT_URL
+
+
+class CommonPackageConfig:
+    """
+    Config class for upstream/downstream packages;
+    this is the config people put in their repos
```

Comment:
Can you please document the overwriting and class inheritance a bit more?

I need to look at classes multiple times to see what is used for what purpose and it can confuse someone that `JobConfig` inherits from the `CommonPackageConfig`.

Reply:
I can try